### PR TITLE
Init: symlink update #135

### PR DESCRIPTION
**Pull request by chervand**
Init: symlink update #135

### CODE REVIEW

File: init [hunk 1]
```diff
@@ -208,6 +208,11 @@ function createSymlink($root, $links) {
         echo "      symlink " . $root . "/" . $target . " " . $root . "/" . $link . "\n";
         //first removing folders to avoid errors if the folder already exists
         @rmdir($root . "/" . $link);
+        //next removing existing symlink in order to update the target
+        $exist = @readlink($root . "/" . $link);
```
Comment (samdark) [inline init]: Do we really need to read contents of what is symlinked?

Comment (chervand) [inline init]: Well, this will preserve modification time. Maybe someone will need this for some reason. :)

Comment (samdark) [inline init]: I mean using `is_link()` seems to be better for the case than `readlink()`.

**Comment by samdark** [comment]
Thanks! https://github.com/yiisoft/yii2-app-advanced/commit/14e4968a70ed24f3b6b2496e408c8e83302f4e56

**Pull request closed without merging**